acoustic_msgs/README.md

```diff
@@ -176,17 +169,20 @@ The image details details how image data is stored.
 
 
 ### Geometry Conventions
-![rx_tx_detail](https://user-images.githubusercontent.com/23006525/160200735-2698b659-27f9-4309-ac3f-1b891858ddb3.png)
+![rx_tx_detail](https://user-images.githubusercontent.com/23006525/167165995-ce578962-be79-4219-89a0-920953ca6cb2.png)
 
-* angles/beam directions in the sptaial domain shall:
-  * be reported as unit vectors according to the in the corridate frame convention specified above.   
+
+* angles/beam directions in the spatial domain shall:
+  * be reported as unit vectors according to the coordinate frame convention specified above.   
   * be reported at time reported in the header timestamp.  (at or before transmit time)  
 * angles/beam directions in the temporal domain shall be reported as transmit (Tx) and receive (Rx) angles according to the diagram above
-  * rx_angle is defined as the angle from the y-z plane at receive time. Positive rotation around the z axis.  This angle defines a cone of possible returns directions for a given beam.
-  * tx_anlge is defined as the angle from the x-y plane at transmit time. Positive rotation around the y axis.  This angle defines a cone that represents the area insonified by the tx pulse.  
+  * rx_angle is defined as the elevation from the X-Z plane at receive time. This elevation is positive toward the Y axis.  This elevation defines a cone of possible return directions for a given return beam.
+  * tx_anlge is defined as the elevation from the Y-Z plane at transmit time. This elevation is positive toward the X axis.  This elevation defines a cone that represents the area insonified by the tx pulse.
+
+
 
 ### Style Conventions
-These messages were designed to comply withe the conventions set forth by the [ROS/Patterens/Conventions guide](http://wiki.ros.org/ROS/Patterns/Conventions#Messages).   Beyond that, we have set the following conventions:
+These messages were designed to comply with the conventions set forth by the [ROS/Patterns/Conventions guide](http://wiki.ros.org/ROS/Patterns/Conventions#Messages).   Beyond that, we have set the following conventions:
 * All vector quantities should have plural names
-* data strutures should be represented as a "structures of arrays" rather than "arrays of structures".
-* vectors quantities may be of length zero.  This shall be interpreted as unreported.  (see internal MSG doccumentaiton for more detials)
+* data structures should be represented as a "structures of arrays" rather than "arrays of structures".
```

Comment:
This is no longer true -- we have beam_unit_vec[] and ping_info[], among others.

Reply:
I don't believe we have ping_info[] anywhere.  I have clarified this a bit in the updated version to specify arrays of core ROS messages .   Let me know what you think.

Reply:
Oh, sorry, I was thinking of `DetectionFlag[] flags`, which is rather annoying w/r/t how much space it takes up in a `rostopic echo`.

Reply:
I forgot about that.  Yeah that one is kind of an edge case.  

I still think we want to say something about how Structure of arrays is preferred since that's how the messages are designed.    

The Detection flag  it is fundamentally a uint8.   We decide to make an exception for this message type since we wanted the enum to be shared for all the messages that use this field.  

anyway,  I took another crack at it.  Take a look.